<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactListSettingsClass</class>
 <widget class="QWidget" name="ContactListSettingsClass">
  <property name="windowTitle">
   <string>ContactListSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>229</width>
           <height>242</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="alternatingCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Alternating colors for background</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showAccountsBox">
            <property name="text">
             <string>Show accounts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="separatorBox">
            <property name="text">
             <string>Hide online/offline separators</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="offlineBox">
            <property name="text">
             <string>Hide offline users</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="clientIconCheckBox">
            <property name="text">
             <string>Show client icons</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hideEmptyBox">
            <property name="text">
             <string>Hide empty groups</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sortstatusCheckBox">
            <property name="text">
             <string>Sort contacts by status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showGroupsBox">
            <property name="text">
             <string>Show groups</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>alternatingCheckBox</tabstop>
  <tabstop>showAccountsBox</tabstop>
  <tabstop>separatorBox</tabstop>
  <tabstop>offlineBox</tabstop>
  <tabstop>clientIconCheckBox</tabstop>
  <tabstop>sortstatusCheckBox</tabstop>
  <tabstop>showGroupsBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
